src/render.jl

```diff
@@ -614,3 +614,14 @@ function text(x::Float64, y::Float64, z::Float64, content::AbstractString)
    )
 end
 
+if (Pkg.installed("Escher") != nothing)
+    import Reactive: Signal
+    import Escher
+
+    """
+    Create a raycaster that triggers on the JavaScript event `event`.
+    """
+    function raycaster(signal::Signal, event::AbstractString = "")
+        Elem(:"three-js-raycaster"; :signal => Escher.makeid((signal, Escher.ToType{Dict{UTF8String, Float64}}())), :event => event)
```

Comment:
this is what I was referring to in the other comment, `makeid` is an implementation detail...